includes/pipeline_page/sidebar.php

```diff
@@ -93,16 +103,17 @@
       <p><a href="<?php echo $pipeline->html_url; ?>/pulls"><?php echo $num_prs; ?></a></p>
     </div>
   </div>
-
-  <div class="border-bottom">
-    <h6>collaborators</h6>
-    <p class="contrib-avatars"><?php echo implode(array_keys($contrib_avatars)); ?></p>
+  <div class="row border-bottom">
+    <div class="col-12">
+      <h6>collaborators</h6>
+      <p class="contrib-avatars"><?php echo implode(array_keys($contrib_avatars)); ?></p>
+    </div>
+  </div>
+  <div>
+    <h6>get in touch</h6>
+      <p><a class="btn btn-sm btn-outline-info" href="https://nfcore.slack.com/channels/<?php echo $pipeline->name; ?>"><i class="fab fa-slack mr-1"></i> Ask a question on Slack</a></p>
+      <p><a class="btn btn-sm btn-outline-secondary" href="<?php echo $pipeline->html_url; ?>/issues"><i class="fab fa-github mr-1"></i> Open an issue on GitHub</a></p>
```

Comment:
```suggestion
    <p><a class="btn btn-sm btn-outline-info" href="https://nfcore.slack.com/channels/<?php echo $pipeline->name; ?>"><i class="fab fa-slack mr-1"></i> Ask a question on Slack</a></p>
    <p><a class="btn btn-sm btn-outline-secondary" href="<?php echo $pipeline->html_url; ?>/issues"><i class="fab fa-github mr-1"></i> Open an issue on GitHub</a></p>
```

Need a code style linter... Been using Black for Python a lot lately and I have fallen in love with it - so nice not having to put any brain power towards this stuff!